### PR TITLE
fix: typos [skip-ci]

### PR DESCRIPTION
**Pull request by metacosm**
**What type of PR is this?**
/kind documentation
[skip ci]

**What does does this PR do / why we need it**:
Fixes a whole bunch of typos (though there are still quite a bit more)

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ Service::
 A service is software that your component links to or depends on.
 Examples of services: MariaDB, MySQL.
 Devfile::
-A portable file responsible for your entire reproducable development environment.
+A portable file responsible for your entire reproductible development environment.
```
Comment (mik-dass) [inline README.adoc:45]: `reproductible` is French I guess, it should be `reproducible`.

Comment (metacosm) [reply README.adoc:45]: Indeed 👍

**Review by mik-dass — APPROVED** [review]
/lgtm

**Pull request merged**